src/constants.js

```diff
@@ -0,0 +1,29 @@
+export const initialState = {
```

Comment:
initial state should in main context file

Reply:
done